### ISSUE TITLE
High cpu and ram %, console spam and freezes

### ISSUE THREAD

**Issue by GiorgioBrux** [Zungrysoft/The-Escher-Dimension#2]
Hi,

I'm playing a save started with the latest release of this datapack (v.1.0.9, released 2 days ago).
However, after a while (can't tell exactly when), the console started to spam
```Trying to add entity with duplicated UUID 0b8d2151-8ea8-4ec8-8dac-74d74329a2af. Existing minecraft:zombie#280597, new: minecraft:zombie#419259```
-like messages, reaching 100k lines so fast that multimc stops logging after a few seconds after the save is opened, and I've got multiple log files 100MB each.
The game keeps having long freezes, where it stops responding, and then works again for a while and repeat.
It was hosted by my friend who has a ryzen 5 3600 and 16GB of RAM, and the cpu % was always around 90% and minecraft also crashed a few times because of OutOfMemory errors, even when allocating 8GB+ to it.

Other saves work fine, and we were just using optifine + xaero's minimap. Using a vanilla instance gives the same result.
If needed here's the 100k lines log [message (21).txt](https://github.com/Zungrysoft/The-Escher-Dimension/files/6836997/message.21.txt) and here's the save [New Wo.zip](https://github.com/Zungrysoft/The-Escher-Dimension/files/6836998/New.Wo.zip)

Sadly I don't have the knowledge to troubleshoot stuff like this, do you have any idea why I am having this issue?

**Comment by Zungrysoft** [comment]
I am not sure what would be causing this. I think it is more likely an issue with Minecraft, since nothing in the datapack ever edits a mob's UUID.

Are you able to run the world for long enough to run `/kill 0b8d2151-8ea8-4ec8-8dac-74d74329a2af` ?

**Comment by GiorgioBrux** [comment]
> Are you able to run the world for long enough to run /kill 0b8d2151-8ea8-4ec8-8dac-74d74329a2af ?

Done, it errors out saying `No entity was found`. Same thing for `/kill 00b7ca03-58c7-4e40-bcfe-bd61de6949f3`
If you are 100% sure it's not the datapack then I apologize and will just try with a new save,  `¯\_(ツ)_/¯`

**Comment by Zungrysoft** [comment]
Sorry, I looked around and I cannot find any reason this should happen. You might also try `/kill @e[type=zombie]`, which won't adversely affect the map in any major way.

I tried opening up the map in an NBT editor, but I cannot find any NBT editor that allows searching for a UUID to delete it from the entity list. And I cannot go through each chunk individually to look for it as that would take way too long.

If the above doesn't solve it, I do not know what to tell you, sorry. I have not heard of anyone else having this specific issue with the map.
